packages-ts/gauntlet-terra-contracts/src/lib/contracts.ts

```diff
@@ -38,26 +45,27 @@ export const loadContracts = (): Contracts => {
         [id]: {
           id,
           abi: getContractABI(id),
-          bytecode: getContractCode(id),
+          bytecode: getContractCode(id, RELEASE_VERSION),
         },
       },
     }
   }, {} as Contracts)
 }
 
-// TODO: Pull it from a Github versioned release artifact
-export const getContractCode = (contractId: CONTRACT_LIST): string => {
-  // Possible paths depending on how/where gauntlet is being executed
-  const possibleContractPaths = [
-    path.join(__dirname, '../..', './artifacts/bin'),
-    path.join(process.cwd(), './artifacts/bin'),
-    path.join(process.cwd(), './packages-ts/gauntlet-terra-contracts/artifacts/bin'),
-  ]
-  const codes = possibleContractPaths
-    .filter((contractPath) => existsSync(`${contractPath}/${contractId}.wasm`))
-    .map((contractPath) => {
-      const wasm = readFileSync(`${contractPath}/${contractId}.wasm`)
-      return wasm.toString('base64')
+export const getContractCode = async (contractId: CONTRACT_LIST, version): Promise<string> => {
+  // get requested release
+  const release = await octokit.request('GET /repos/{owner}/{repo}/releases/tags/{tag}', {
+    owner: 'smartcontractkit',
+    repo: 'chainlink-terra',
+    tag: version,
+  })
+
+  const codes = release.data.assets
```

Comment:
This probably needs to be a `await Promise.all()` as the mapping returns a promise